In [1]:
import math

Step 1: Guessing 

Guess 1: Maybe the group's blood pressure is about the same as most people, which is 120 mm Hg.
Guess 2: But maybe, just maybe, the group's blood pressure is not the same. Maybe it's different in a special way.

Step 2:
We use a special rule to see if our guesses are right. This rule has two parts:

Part 1: One guess is that blood pressure is normal.
Part 2: The other guess is that it's different.

In [2]:
# Given values
sample_mean = 130.1
population_mean = 120
sample_std_dev = 21.21
sample_size = 100

In [3]:
# Calculate the test statistic
test_statistic = (sample_mean - population_mean) / (sample_std_dev / math.sqrt(sample_size))
print("Calculated test statistic:", test_statistic)

Calculated test statistic: 4.761904761904759


Understanding the Result:

We did some math to see if a group's blood pressure was really different from what's normal. We got 4.76190. Imagine this number as a signal. If it's smaller than a certain level, things are okay. If it's bigger, something might be up.

In our case, the signal (4.76190) isn't as big as the level it needs to be to worry. So, even though the blood pressure is a bit different, it's not different enough to be a big concern.

Think of it like measuring how high a friend can jump. If they jump a bit higher than usual but not over the really high line on the wall, it's all good. Our math helps us decide if things are just a small change or something more important.